### PR TITLE
[WIP] New command start

### PR DESCRIPTION
**Pull request by michal-franc**
### New Command start

To use:
```
gopom start <task_name>
```
This is still super messy code which assumes that you have file called `timer.mp3` at the place where you run the app (i will send timer.mp3 in slack as i don't want to upload the file for now to repo - licensing, size, compression).

Libraries used:
- github.com/faiface/beep - for playing sounds
- github.com/cheggaaa/pb/v3 - for creating progress bar

#### How does it look - so far?
image

### Things to do next:
- app will crash if there is no file `timer.mp3` to play
- app will crash if there is no arg provided to `start` task
- the sound will stop after first loop
- there is no proper cleaning and handling of resources when doing SIGINT

### CODE REVIEW

File: internal/app/gopom/commands/start.go [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+		}
+		defer streamer.Close()
+
+		done := make(chan bool)
+		speaker.Play(beep.Seq(streamer, beep.Callback(func(){
+			done <- true
+		})))
+
+		// 25 minutes
+		seconds := 25 * 60
```
Comment (BartoszCoyote) [inline internal/app/gopom/commands/start.go]: //TODO move to an external config file

File: internal/app/gopom/commands/start.go [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+		tmpl := `{{ "Working on task:" }} {{ string . "task" | green }} {{ bar . "<" "-" (cycle . "↖" "↗" "↘" "↙" ) "." ">"}} {{string . "timer" | green}}`
```
Comment (tomekbielaszewski) [inline internal/app/gopom/commands/start.go]: feature idea: how about moving templates to seperate file and make it plugin'able

File: internal/app/gopom/commands/start.go [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+		// start bar based on our template
+		bar := pb.ProgressBarTemplate(tmpl).Start(seconds)
+		// set values for string elements
+		bar.Set("task", taskName).
+			Set("timer", fmtTimer(0))
+
+		for i := 0; i < seconds; i++ {
```
Comment (tomekbielaszewski) [inline internal/app/gopom/commands/start.go]: I imagined this as concurrent process which triggers some listeners every now and then

File: internal/app/gopom/commands/start.go (hunk 1 above)
Comment (tomekbielaszewski) [reply internal/app/gopom/commands/start.go]: Have we integrated viper?

**Pull request merged**